docs/solutions/linux/scripts/linux_ubuntu_fleetd_healthcheck.sh

```
@@ -0,0 +1,322 @@
+# (usually empty; osqueryd's stdout/stderr goes to syslog/journal by default — see below)
+collect_log "osquery logs" "/var/log/osquery" "${WORK_DIR}/logs/osquery"
+
+# osquery filesystem logger output — only populated when logger_path/logger_plugin
+# is set to "filesystem" in agent options. This is where result/status logs
+# (osqueryd.INFO*, osqueryd.results.log, osqueryd.snapshots.log, etc.) actually live.
+collect_log "osquery filesystem logger" "/opt/orbit/osquery_log" "${WORK_DIR}/logs/osquery_log"
+
+# systemd journal for orbit.service (last 500 lines)
+if command -v journalctl >/dev/null 2>&1; then
+  journalctl -u orbit.service --no-pager -n 500 \
+    > "${WORK_DIR}/logs/orbit_journal.log" 2>&1 && \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)